### ISSUE TITLE
Джаспер и Кассия . Реакция на атаку.

### ISSUE THREAD

**Issue by Kor-Angar** [dosinabox/g2nr_unofficial_update#105]
члены гильдии воров неадекватно отреагировали на агрессию героя. 

проигнорировали её и занимались своими делами.

https://youtu.be/OzduAYROvw0?t=1909

**Comment by dosinabox** [comment]
Тут нужно полноценную систему вводить, чтобы за "своих" заступались. Гильдия воров, Бромор и шлюхи, бандиты против ГГ в бандитских доспехах, Лобарт/Бенгар/Секоб/Акил и их фермеры, мужья и жены. 
А ворам, наверное, можно делать перманентную враждебность, если ГГ кого-то атаковал. И квесты их проваливать.

**Comment by Kor-Angar** [comment]
> А ворам, наверное, можно делать перманентную враждебность, если ГГ кого-то атаковал. И квесты их проваливать.

раньше Кассия "слышала" если Герой атаковал Рамираза и атаковала героя. а сейчас не услышала.  может быть глюк, а может быть нет :-)

надо будет перепроверить.

**Comment by dosinabox** [comment]
> раньше Кассия "слышала" если Герой атаковал Рамираза и атаковала героя.

Допустим, услышала. Допустим, атаковала ГГ, заступаясь за Рамиреза. Допустим, ГГ не стал драться, убежал и вернулся через день. Как должны реагировать на него воры в таком случае? Могут ли они ему доверять, обучать, давать новые задания и закрывать текущие дела?

**Comment by Kor-Angar** [comment]
> Как должны реагировать на него воры в таком случае? 

атаковать героя

> Могут ли они ему доверять, обучать, давать новые задания и закрывать текущие дела?

нет. у них же правила такие. наказание одно.. смерть :-)

**Comment by dosinabox** [comment]
Или ещё ситуация: ГГ ударил Рамиреза и ушел из логова на несколько часов. Должен ли Рамирез рассказать Кассии и Джесперу о том, что на него напали, с соответствующими изменениями их отношения?

Должны ли воры на поверхности (Халвор, Джо, садовник, Ренгару, ) со временем узнавать о том, что ГГ устроил драку в логове или убил кого-то из них? Как они должны вести себя в таком случае?

**Comment by Kor-Angar** [comment]
> Или ещё ситуация: ГГ ударил Рамиреза и ушел из логова на несколько часов. Должен ли Рамирез рассказать Кассии и Джесперу о том, что на него напали, с соответствующими изменениями их отношения?

конечно. у них же правила. нельзя обнажать оружие.  ГГ ударил любого, все нападают на ГГ.

> Должны ли воры на поверхности (Халвор, Джо, садовник, Ренгару, ) со временем узнавать о том, что ГГ устроил драку в логове или убил кого-то из них? Как они должны вести себя в таком случае?

наверное никак. это уже будет слишком сложно. опять же они не бывают в канализацию. только хозяйка гостиницы узнаёт, а она нападает на героя и погибают.

**Comment by Kor-Angar** [comment]
можно в случае сдачи квеста по гильдии воров Андре отправить в канализацию кого нибудь из ментов. Например Микки и еще одного безымянного мента который днём тренируется.
